heapster-saw/src/Verifier/SAW/Heapster/Implication.hs

```diff
@@ -5402,6 +5410,27 @@ proveVarLLVMBlocks' x ps psubst mb_bps_in mb_ps = case mbMatch mb_bps_in of
       implSwapInsertConjM x (Perm_LLVMBlock bp') ps_out' 0
 
 
+  -- If proving an opaque named shape, the only way to prove the memblock
+  -- permission is to have it on the left, but we don't have a memblock
+  -- permission on the left with this exactly offset, length, and shape, because
```

Comment:
Grammar nitpick: Should be "exact offset, length, ..." instead of "exactly offset, length, ..."